livereveal/main.js

```diff
@@ -106,8 +115,22 @@ function markupSlides(container) {
     
     // Put .end_space back at the end after all the rearrangement
     $('.end_space').appendTo('div#notebook-container');
+    return selected_cell_slide;
 }
 
+// Set the #slide-x-y part of the URL to control where the slideshow will start
+function setStartingSlide(selected) {
+    var start_slideshow = config.get_sync('start_slideshow_at');
+    if (start_slideshow === 'selected') {
+        // Start from the selected cell
+        window.location.hash = "/slide-"+selected[0]+"-"+selected[1];
```

Comment:
I just tried doing it this way, but I didn't like the result - the last visible slide is very clearly on the screen for a moment after going into slideshow mode before it changes to the cell you want, whereas the hash-changing method in this PR really makes it start at the desired slide.